runtime/browser/xwalk_presentation_service_delegate_android.h

```diff
@@ -1,37 +1,21 @@
-// Copyright (c) 2015 Intel Corporation. All rights reserved.
+// Copyright (c) 2016 Intel Corporation. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
 #ifndef XWALK_RUNTIME_BROWSER_XWALK_PRESENTATION_SERVICE_DELEGATE_ANDROID_H_
 #define XWALK_RUNTIME_BROWSER_XWALK_PRESENTATION_SERVICE_DELEGATE_ANDROID_H_
 
 #include <string>
-#include <utility>
-
-#include "base/containers/scoped_ptr_hash_map.h"
-#include "base/memory/scoped_ptr.h"
-#include "base/memory/weak_ptr.h"
-#include "content/public/browser/presentation_service_delegate.h"
-#include "content/public/browser/web_contents_user_data.h"
-
-namespace content {
-class WebContents;
-}  // namespace content
+#include "xwalk/runtime/browser/xwalk_presentation_service_delegate.h"
 
 namespace xwalk {
 
-class PresentationFrame;
-class PresentationSession;
-class XWalkBrowserContext;
-
 class XWalkPresentationServiceDelegateAndroid
-    : public content::WebContentsUserData
-        <XWalkPresentationServiceDelegateAndroid>,
-      public content::PresentationServiceDelegate,
+    : public XWalkPresentationServiceDelegate,
+      public content::WebContentsUserData<
+        XWalkPresentationServiceDelegateAndroid>,
```

Comment:
BTW, one thing you can do is use `clang-format` to format your C++ code automatically:

```sh
cd /path/to/src/xwalk
CHROMIUM_BUILDTOOLS_PATH=/path/to/src/buildtools clang-format -i foo/bar.cc
```

It's possible it will reformat more than the code you touched. You can then either accept all changes or get rid of those unrelated to your patch.